### PR TITLE
DOC: animation faster

### PR DESCRIPTION
**Pull request by jklymak**
## PR Summary

The new animation tutorial takes over 30s to render in sphinx gallery on my machine.  That seems unnecessary for didactic purposes.   This version takes less than 5 s.  Maybe someone wants different aesthetics, but this really is pretty slow.  

@chahak13 I think you wrote the original.  happy to defer to your interest in how this looks, but I still strongly encourage that it be _much_ faster.  

## PR Checklist

**Documentation and Tests**
- [ ] Has pytest style unit tests (and `pytest` passes)
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] New plotting related features are documented with examples.

**Release Notes**
- [ ] New features are marked with a `.. versionadded::` directive in the docstring and documented in `doc/users/next_whats_new/`
- [ ] API changes are marked with a `.. versionchanged::` directive in the docstring and documented in `doc/api/next_api_changes/`
- [ ] Release notes conform with instructions in  `next_whats_new/README.rst` or `next_api_changes/README.rst`

### CODE REVIEW

**Review by tacaswell — APPROVED** [review]

**Review by timhoffm — COMMENTED** [review]
Reducing the number of points is reasonable. However, with that, you only plot in a tiny fraction of the Axes, and the (somewhat complex) x-y relation does not become apparent.
I suggest to change the data as well, e.g. plot 36 points on a unit circle.

**Comment by jklymak** [comment]
OK, changed this much more to something physical if trivial.  Also did things like label the axes etc...

File: tutorials/introductory/animation_tutorial.py [hunk 1]
```diff
@@ -87,32 +87,37 @@
...
+line2 = ax.plot(t[0], z2[0], label=f'v0 = {v02} m/s')[0]
+ax.set(xlim=[0, 3], ylim=[-4, 10], xlabel='Time [s]', ylabel='Z [m]')
+ax.legend()
 
 
 def update(frame):
-    # .set_offsets replaces the offset data for the entire collection with
-    # the new values. Therefore, to also carry forward the previously
-    # calculated information, we use the data from the first to the current
-    # frame to set the new offsets.
```
Comment (chahak13) [inline tutorials/introductory/animation_tutorial.py:107]: I feel like an explanation about why we take the entire data for the new frame should be there since this is a tutorial. Maybe reword it a little but not remove it completely?

**Comment by chahak13** [comment]
The original idea was to not have something entirely trivial to show how to go about doing slightly more problem based, which is why I did not go for, say a unit circle. I'm happy with the projectile motion example that you added if it's making the doc load much faster :)

File: tutorials/introductory/animation_tutorial.py (hunk 1 above)
Comment (jklymak) [reply tutorials/introductory/animation_tutorial.py:107]: I changed it a bit.  Note I also added comments for the steps where the data was added, which basically says what you are saying in the old text, so I think it's relatively clear?  If not, happy to take suggestions.

**Review by ksunden — APPROVED** [review]

**Comment by jklymak** [comment]
Thanks @ksunden 

I think this was important to get in to keep the doc build reasonable.  OTOH, @chahak13 or @story645  you are more than welcome to open a new PR to tweak the wording.
